container/raw/handler.go

```diff
@@ -221,7 +221,12 @@ func (self *rawContainerHandler) getFsStats(stats *info.ContainerStats) error {
 }
 
 func (self *rawContainerHandler) GetStats() (*info.ContainerStats, error) {
-	stats, err := libcontainer.GetStats(self.cgroupManager, self.rootFs, os.Getpid(), self.ignoreMetrics)
+	pid := 0
+	// Do not collect network stats for non root cgroups.
+	if self.name == "/" {
```

Comment:
Nit: this will be the third instance of ```if self.name == "/" {``` check.  Maybe create a isRootCgroup() to help make this uniform and clear?